### PR TITLE
Add tutorials for Avalonia UI

### PR DESCRIPTION
**Pull request by CreateLab**
I add new useful tutorials for AvaloniaUI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,12 @@ Contributions are always welcome! Please take a look at the [contribution guidel
 - [GroupMe Desktop Client](https://github.com/alexdillon/GroupMeClientAvalonia) - GroupMe Desktop Client Avalonia is an open-source, cross platform, modular client for GroupMe messenging.
 
 ## Tutorials
-
+- [Quickstart](https://avaloniaui.net/docs/quickstart/) - Info for quick start with Avalonia UI.
+- [AvaloniaUI Tutorial](https://avaloniaui.net/docs/tutorial/) - Official AvaloniaUI tutorial.
+- [Avalonia for WPF Developers](https://avaloniaui.net/docs/quickstart/from-wpf) - Quick start for WPF developers.
```
Comment (FoggyFinder) [inline README.md]: Three of links are refer to official documentation. So it's a bit unclear why have you decided to split them up?

Comment (CreateLab) [reply README.md]: Yes I understand  that there are links on same resource. But as for my opinion it's the most important part, and avalonia users needs short links to important parts of avalonia tutorial

File: README.md [hunk 2]
```diff
@@ -54,7 +54,12 @@ Contributions are always welcome! Please take a look at the [contribution guidel
...
+- [Saving Routing State with ReactiveUI and Avalonia](https://habr.com/en/post/462307/) - For better UX, your app should be capable of saving state to the disk when the app is suspending and of restoring state when the app is resuming.
```
Comment (OmidID) [inline README.md]: I think it's better to move this to Article section. Do you think same?

Comment (CreateLab) [reply README.md]: As I think - no, cause that articles are tutorials

**Pull request merged**